Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -338,6 +339,13 @@ void ProjectData::parseProcesses(BaseLib::ConfigTree const& processes_config,
                         "dimension");
             }
         }
+        else if (type == "DensityDrivenFlow")
```

Comment:
yes, this settles any confusion. Will it be then HTC with the option to switch from HTC to HC / HT or we keep 3 separates implementations?